packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -371,9 +377,28 @@ export const getCacheUpdateMatrix = (customerId: string | null) => {
         ): CacheUpdateMatrixElement => {
             const basketId = params.parameters.basketId
 
+            const updateBasketQuery = {
+                update: [
+                    {
+                        name: 'basket',
+                        key: ['/customers', customerId, '/baskets', {customerId}],
+                        updater: (oldData: any) => {
+                            const newBaskets = oldData.baskets.map((basket: any) => {
+                                const isTargetBasket = basket.basketId === basketId
+                                return basket.basketId === basketId ? response : basket
+                            })
+                            return {
+                                ...oldData,
+                                baskets: newBaskets
+                            }
+                        }
+                    }
+                ]
+            }
+
             return {
-                ...updateBasketQuery(basketId),
-                ...invalidateCustomerBasketsQuery(customerId)
+                ...updateBasketQuery
+                // ...invalidateCustomerBasketsQuery(customerId)
```

Comment:
No need to refetch since we already get the newest basket from this mutation. We just need to update the cache accordingly, this will save us one network call.

Reply:
I don't see why we should keep this response in a different cache as what we did previously. It is likely that we will have any query that reuses this cache at all, meaning, this cache is stored, but never consumed. 
````
update: [
                      {
                          name: 'basket',
                          key: ['/baskets', basketId, arg],
                          updater: () => response
                      }
                  ]
```
The basket mutation hook is supposed to modify data in the customer baskets, which stored in `'/customers', customerId, '/baskets', {customerId}` and most UI and data usage is from this cache.